app/navigation/navigator.py

```diff
@@ -20,8 +26,23 @@ def _valid_destination(self, destination):
         :returns: True if valid, False otherwise
```

Comment:
Sorry Dan, missed your comment when I merged. I'll make the missed changes in a new PR